src/main/java/com/microsoftopentechnologies/windowsazurestorage/WAStoragePublisher.java

```diff
@@ -435,9 +435,15 @@ public synchronized void perform(
                     zipArchiveBlob = serviceData.getArchiveBlobs().get(0);
                 }
 
-                run.addAction(new AzureBlobAction(expContainerName, expShareName, storageType,
-                        serviceData.getIndividualBlobs(), zipArchiveBlob, allowAnonymousAccess,
-                        getStorageCredentialId()));
+                List<AzureBlob> individualBlobs = serviceData.getIndividualBlobs();
```

Comment:
is this  azure call? if yes, pls add error handling.

Reply:
no